app/views/scopes/card.rb

```diff
@@ -0,0 +1,30 @@
+# auto_register: false
+# frozen_string_literal: true
+
+module Site
+  module Views
+    module Scopes
+      class Card < Hanami::View::Scope
```

Comment:
Can see the value of having a proper class here in some situations, but for many – maybe most – it’d be an identical implementation. Do we need the concrete class always?

Reply:
@makenosound since this kind of slot functionality is open-ended, I could just shift it into the base scope class. This would mean you only have to write partials, not custom classes. I'll do that when I get a moment!

Reply:
Woop, that’d be amazing.

Reply:
@makenosound this is done now :) Overall diff is much smaller. Take another look and let me know how that feels.